### ISSUE TITLE
cert signed by happy hacking ca although its public beta and i got a beta invite also

### ISSUE THREAD

**Issue by Thumpxr** [alexalouit/ISPConfig-letsencrypt#14]
I have some issues, as i think letsencrypt doesnt use the right server, although its configured in the cli.ini to their acme server.

/etc/letsencrypt/cli.ini content:

```
# This is an example of the kind of things you can do in a configuration file.
# All flags used by the client can be configured here. Run Let's Encrypt with
# "--help" to learn more about the available options.

# Use a 4096 bit RSA key instead of 2048
rsa-key-size = 4096

# Always use the staging/testing server
#server = https://acme-staging.api.letsencrypt.org/directory

# Uncomment and update to register with the specified e-mail address
# email = foo@example.com

# Uncomment to use a text interface instead of ncurses
# text = True

# Uncomment to use the standalone authenticator on port 443
# authenticator = standalone
# standalone-supported-challenges = dvsni

# Uncomment to use the webroot authenticator. Replace webroot-path with the
# path to the public_html / webroot folder being served by your web server.
# authenticator = webroot
# webroot-path = /usr/share/nginx/html

text = True
agree-tos = True
authenticator = webroot
server https://acme-v01.api.letsencrypt.org/directory
```

**Comment by alexalouit** [comment]
Are you sure domain is whitelisted by Let's Encrypt?

**Comment by Thumpxr** [comment]
I am. They send me a mail on 16.11.15 and whitelisted all my domains.

Date: Wed, 9 Dec 2015 03:33:49 -0800

**Comment by alexalouit** [comment]
Certificate has he already been generated with the test server?
If this was done, it is necessary to remove it (remove `/etc/letsencrypt/accounts/DOMAIN`, `/etc/letsencrypt/archive/DOMAIN` and `/etc/letsencrypt/renewal/DOMAIN`)

Otherwise, a good way to debug is to run the code by hand.
Even if the final certificate is delivred, this will not affect ISPConfig, simply uncheck+save and check+save Let's Encrypt field.

debug command:
`/root/.local/share/letsencrypt/bin/letsencrypt auth -a webroot --email postmaster@DOMAIN --domains DOMAIN --webroot-path /var/www/DOMAIN/web -vvvvvvv`

**Comment by MZorzy** [comment]
Thumpxr , u miss a =
server https://acme-v01.api.letsencrypt.org/directory -> 
server = https://acme-v01.api.letsencrypt.org/directory

BUT now on open beta --server is deprecated

**Comment by alexalouit** [comment]
Oh, I missed that, thank's. fix in 90d6eb0279b1e81959cf34ef1bb72ae5ffbe2599

**Comment by Thumpxr** [comment]
Thanks. Solved.

Date: Wed, 9 Dec 2015 05:43:31 -0800
